integration-tests/fake_spine/fake_spine/pystache_message_builder.py

```diff
@@ -0,0 +1,50 @@
+"""A module that defines a Pystache-based MessageBuilder."""
```

Comment:
I copied this file over from integration-adaptors-common. Not idea I know. The fake-spine project is setup is currently independent of the rest of the code base. We'd need to change the Dockerfile and some of the build to bring in other modules as a dependency.